index.html

```diff
@@ -37,34 +36,206 @@ <h2 class="logo">My Logo</h2>
+        <li class="small-langli">css</li>
+        <li class="small-langli">javascript</li>
+      </ul>
+      <div>
+        <button class="seeproject active">See Project</button>
+      </div>
+    </section>
+    <br />
+    <section id="tonic2" class="work-cards">
+      <div class="work-img"><img class="work-img" src="icons/tonic2.png" alt="" /></div>
+      <h2 id="h-tag" class="tonic">Tonic</h2>
+      <div>
+        <ul class="canopy-list">
+          <ol class="canopy">
+            CANOPY
+          </ol>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <div class="aboutme-intro">
+        <p id="p-tag">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <ul class="small-langul">
+        <li class="small-langli">html</li>
+        <li class="small-langli">css</li>
+        <li class="small-langli">javascript</li>
+      </ul>
+      <div>
+        <button class="seeproject active">See Project</button>
+      </div>
+    </section>
+    <br />
+    <section id="multi2" class="work-cards">
+      <div class="work-img"><img class="work-img" src="icons/multipost2.png" alt="" /></div>
+      <h2 id="h-tag" class="multi">Multi-Post Stories</h2>
+      <div>
+        <ul class="canopy-list">
+          <ol class="canopy">
+            CANOPY
+          </ol>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <div class="aboutme-intro">
+        <p id="p-tag">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <ul class="small-langul">
+        <li class="small-langli">html</li>
+        <li class="small-langli">css</li>
+        <li class="small-langli">javascript</li>
+      </ul>
+      <div>
+        <button class="seeproject active">See Project</button>
+      </div>
+    </section>
+    <br />
+    <section class="aboutme">
+      <h2 id="h-tagaboutme">About Myself</h2>
+      <div class="aboutme-intro">
+        <p id="p-tag1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </div>
+      <div class="socials">
+        <h4 class="letsconnect">LET'S CONNECT</h4>
+        <ul class="socials-mainlist">
+          <ol class="socials-list">
+            <a href="#"><img src="./icons/github.png" alt="github icon" /></a>
+          </ol>
+          <ol class="socials-list">
+            <a href="#"><img src="./icons/linkedin.png" alt="linkedin icon" /></a>
+          </ol>
+          <ol class="socials-list">
+            <a href="#"><img src="./icons/twitter.png" alt="twitter icon" /></a>
+          </ol>
+          <ol class="socials-list">
+            <a href="#"><img src="./icons/medium.png" alt="medium icon" /></a>
+          </ol>
+        </ul>
```

Comment:
As the previous reviewer suggested, please replace the `<ol>` tags in this section with `<li>` tags🙏. This improves the semantics of your webpage